osv/models.py

```diff
@@ -30,7 +30,7 @@
 from . import sources
 from . import vulnerability_pb2
 
-SCHEMA_VERSION = '1.3.0'
+SCHEMA_VERSION = '1.3.1'
```

Comment:
```suggestion
SCHEMA_VERSION = '1.4.0'
```

Reply:
Updated again to 1.4.0

Reply:
We don't actually support 1.4.0 properly yet. I created https://github.com/google/osv.dev/issues/1093 to track the needed changes.